features/step_definitions/cookie_steps.rb

```diff
@@ -0,0 +1,10 @@
+Then /^I should receive a "([a-z0-9_]+)" cookie which is "([a-zA-Z]+)"$/ do |cookie_name, cookie_property|
+  headers = @response.headers[:set_cookie].select do |h|
```

Comment:
I've stopped using rest-client so I'll see if there's a nicer way to do this with `Capybara::Session`. Maybe [show_me_the_cookies](https://github.com/nruth/show_me_the_cookies).